requirements.txt

```diff
@@ -24,7 +24,7 @@ Flask-Mail==0.9.1
 flask-marshmallow==0.12.0
 Flask-Migrate==2.5.2
 Flask-Principal==0.4.0
-Flask-Security==3.0.0
+Flask-Security-Too==3.2.0
```

Comment:
@tuz666 do we expect a performance boost from the bump? Did you deploy this branch on dev?

Reply:
Yes, you are right, both version uplift 3.1.0 and 3.2.0 promise a performance boost, and only that little relationship change needed. I tried to create the DB migration script but it turned out this modification won't cause any direct DB change, also we had a conversation about the topic with @pvj and he confirmed this too.

Reply:
Thanks!

> Did you deploy this branch on dev?

And how about this? I'd do this first before merging.

Reply:
I deployed it on the dev now, not sure about the situation of the frontend staging, I reach out to you directly.